### DISCUSSION TITLE
Double click on media controls re-centers it

### DISCUSSION THREAD

**Discussion by bryceorchard** [iina/iina#4564] [Ideas]
Very simple feature request,
Since you can drag the media controls around, and I like things tidy (as many ppl do), I thought that double clicking in the space between icons of the media controls panel (screenshot attached) should reset it to its original position.
<img width="459" alt="CleanShot 2023-08-06 at 22 59 42@2x" src="https://github.com/iina/iina/assets/92669537/91acd202-793f-4a19-af2c-e35fe58b40fb">

**Comment by low-batt** [comment]
I've entered issue #4565 for this. See that issue for more discussion.
